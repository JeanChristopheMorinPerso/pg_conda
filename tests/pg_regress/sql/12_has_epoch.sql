BEGIN;
CREATE EXTENSION pg_conda;

SELECT conda_has_epoch('1.0.0') = FALSE;
SELECT conda_has_epoch('1.0.0'::condaversion) = FALSE;
SELECT conda_has_epoch('2!1.0.0') = TRUE;
SELECT conda_has_epoch('2!1.0.0'::condaversion) = TRUE;
ROLLBACK;
