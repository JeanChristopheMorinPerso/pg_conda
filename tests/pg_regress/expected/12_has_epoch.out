BEGIN;
CREATE EXTENSION pg_conda;
SELECT conda_has_epoch('1.0.0') = FALSE;
 ?column? 
----------
 t
(1 row)

SELECT conda_has_epoch('1.0.0'::condaversion) = FALSE;
 ?column? 
----------
 t
(1 row)

SELECT conda_has_epoch('2!1.0.0') = TRUE;
 ?column? 
----------
 t
(1 row)

SELECT conda_has_epoch('2!1.0.0'::condaversion) = TRUE;
 ?column? 
----------
 t
(1 row)

ROLLBACK;
